### ISSUE TITLE
Questie' tried to call the protected function 'CompactRaidFrame11:SetAttribute()

### ISSUE THREAD

**Issue by rbpaiett** [Questie/Questie#2846]
## Bug description

This appeared during a raid in UBRS when the raid group icons updated, hard to reproduce because it's in a raid:

1x [ADDON_ACTION_BLOCKED] AddOn 'Questie' tried to call the protected function 'CompactRaidFrame11:SetAttribute()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML\CompactUnitFrame.lua"]:163: in function `CompactUnitFrame_SetUnit'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:318: in function `CompactRaidFrameContainer_AddUnitFrame'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:254: in function `CompactRaidFrameContainer_AddPlayers'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:176: in function `CompactRaidFrameContainer_LayoutFrames'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:130: in function `CompactRaidFrameContainer_TryUpdate'
[string "@Blizzard_CompactRaidFrames\Blizzard_CompactRaidFrameContainer.lua"]:57: in function `CompactRaidFrameContainer_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

## Questie version

Ver. 6.3.12

**Comment by rbpaiett** [comment]
This seems to continue on with 6.3.15.  I had this together with poisoner and missingtradeskilllist.  First the error fell on poisoner.  Then I removed poisoner and it fell on missingtradeskillist.  Then I removed missingtradeskilllist and the error fell on Questie.  Then I removed questie and it never appeared again.

It appears to happen when there's a change in status of a group member when in a dungeon group (if they level, if they leave, etc.) 

This was the error in 6.3.15:

1x [ADDON_ACTION_BLOCKED] AddOn 'Questie' tried to call the protected function 'CompactRaidFrame2:Show()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `Show'
[string "@FrameXML\CompactUnitFrame.lua"]:359: in function `CompactUnitFrame_UpdateVisible'
[string "@FrameXML\CompactUnitFrame.lua"]:292: in function `CompactUnitFrame_UpdateAll'
[string "@FrameXML\CompactUnitFrame.lua"]:125: in function <FrameXML\CompactUnitFrame.lua:53>

This happened when we were in Mana Tombs.

Hope this further information is useful.
